MetriCam2.Controls/CameraSettingsControl.cs

```diff
@@ -647,5 +644,11 @@ private TextBox CreateTextBox(Camera.ParamDesc paramDesc, int currentRow)
             return textBoxValue;
         }
         #endregion
+
+        private Point2i StringToPoint2i(string s)
```

Comment:
I'd name it `ResolutionToPoint2i` to clarify that it's different from `Point2i.Parse`

Reply:
Renamed. I'll move it to the Conversions helper class in the Attributes-branch. Moving all this code around seems off topic here.

Reply:
OK. I just wanted to squeeze them in as fast as possible :D